src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
Kindly make sure that the `App.js` component is class-based as per project requirements. Although you'll eventually refactor your components to using Hooks, it is important to learn how to use both class and functional components. For now, you'll be learning how to use class-based components until further notice.